images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -284,6 +284,60 @@ meteringconfig_storage_s3Compatible_create_secret: "{{ _storage_spec | json_quer
 meteringconfig_storage_s3Compatible_credentials_secret_name: "{{ _storage_spec | json_query('hive.s3Compatible.secretName') }}"
 meteringconfig_storage_s3Compatible_endpoint: "{{ _storage_spec | json_query('hive.s3Compatible.endpoint') }}"
 
+#
+# Override the default values to handle Openshift, and non-Openshift configurations
+#
+meteringconfig_ocp_disabled: "{{ lookup('env', 'DISABLE_OCP_FEATURES') | bool or meteringconfig_default_values | combine(meteringconfig_spec_overrides, recursive=True) | json_query('disableOCPFeatures.enabled') }}"
+_meteringconfig_ocp_disabled_overrides: "{{ meteringconfig_ocp_disabled | ternary(_ocp_disabled_overrides, _ocp_enabled_overrides) }}"
+
+_meteringconfig_ocp_prometheus_default_url: "https://prometheus-k8s.openshift-monitoring.svc:9091/"
```

Comment:
We only seem to use this once, maybe we should just set this directly in `_ocp_enabled_overrides`